database/factories/FlightFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Lightit\Backoffice\Cities\Domain\Models\City;
+use Lightit\Backoffice\Flights\Domain\Models\Flight;
+
+/**
+ * @extends Factory<Flight>
+ */
+class FlightFactory extends Factory
+{
+    protected $model = Flight::class;
+    public function definition(): array
+    {
+        $departureTime = $this->faker->dateTimeBetween('now', '+2 months');
+        return [
+            'origin_id'=> City::inRandomOrder()->value('id'),
+            'destination_id'=> City::inRandomOrder()->value('id'),
+            'airline_id'=> City::inRandomOrder()->value('id'),
```

Comment:
We shouldn't use existing records; we must use factories for every related model. For example:


```suggestion
            'origin_id'=> CityFactory::new(),
```

Reply:
+1

Reply:
Changed them all to use their respective factories